pkg/filters/binary.go

```diff
@@ -0,0 +1,182 @@
+package filters
+
+import (
+	"encoding/binary"
+	"strconv"
+	"strings"
+	"unsafe"
+
+	bpf "github.com/aquasecurity/libbpfgo"
+	"github.com/aquasecurity/tracee/pkg/utils/proc"
+)
+
+type nsBinary struct {
+	mntNS uint32
+	path  string
+}
+
+type BinaryFilter struct {
+	equal     map[nsBinary]bool
+	notEqual  map[nsBinary]bool
+	hostMntNS uint32
+	enabled   bool
+}
+
+func getHostMntNS() uint32 {
+	if ns, err := proc.GetProcNS(1, "mnt"); err == nil {
+		return uint32(ns)
+	}
+
+	return 0
```

Comment:
If there was an error, is this a valid value? There is no later logic taking this value into concern, so if it's not necessarily correct we should probably add an error return value here.

Reply:
Done